### PR TITLE
Update Node.js to v0.12.12

### PR DESCRIPTION
**Pull request by fhemberger**
Adds latest OpenSSL security updates

### CODE REVIEW

File: cross/node/digests [hunk 1]
```diff
@@ -1,3 +1 @@
-node-v0.12.2.tar.gz SHA1 a969f17a0a6c9238584f8946d96e8d39be8eb957
-node-v0.12.2.tar.gz SHA256 ac7e78ade93e633e7ed628532bb8e650caba0c9c33af33581957f3382e2a772d
-node-v0.12.2.tar.gz MD5 b7f4a9f2e361a7026789a7d6c45a6d30
+node-v0.12.12.tar.gz SHA256 61e4c176fd882498778b1a3907a5fe5c9e95e6cc8438b0d053d953aed3620273
```
Comment (Dr-Bean) [inline cross/node/digests]: Please create digests with `make digests` in the cross package dir.

File: spk/node/Makefile [hunk 2]
```diff
@@ -1,6 +1,6 @@
 SPK_NAME = node
-SPK_VERS = 0.12.2
-SPK_REV = 5
+SPK_VERS = 0.12.12
+SPK_REV = 1
```
Comment (Dr-Bean) [inline spk/node/Makefile]: `SPK_REV` goes up for every new package release. It never resets.

**Comment by Dr-Bean** [comment]
Thanks. I thought the 0.12 line was end-of-life, apparently not.
Have you tested that compiling and running node works as expected?

**Comment by fhemberger** [comment]
Fixed the issues you've mentioned. Node.js v0.12 is end-of-life in December 2016 (together with the OpenSSL version it uses). Between v0.12.2 and v0.12.12, there have been a number of important security fixes, so it's highly recommended to update it.

As long as gcc 4.7 on Synology prevents #2192 to land, I think it's the best you can do at the moment. DSM6 will probably solve this issue.

I compiled it using your Docker container, but haven't ran further tests.

**Comment by Dr-Bean** [comment]
Can be merged once tested that `node` and `npm` command work on a non-x86 based device. I assume so, but we've had issues with that in the past.

**Comment by fhemberger** [comment]
Update:
- **Node.js 0.12.13 (Security release)**
- Use new official Node.js logo

**Comment by Dr-Bean** [comment]
See the other issue: this one will be closed as well.

**Comment by fhemberger** [comment]
@Dr-Bean I'm fine with that, but please remove node soon from the repo.

**Pull request closed without merging**